docs/changelog.rst

```diff
@@ -18,6 +18,10 @@ Bug fixes:
 * Improved naming of temporary files by separating the random part with the file extension.
 * Fixed the ``auto`` value for the :ref:`reflink` config option.
 * Fixed lyrics plugin only getting part of the lyrics from ``Genius.com`` :bug:`4815`
+* Album flexible fields are now correctly saved. For instance MusicBrainz external links
+  such as `bandcamp_album_id` will be available on albums in addition to tracks.
+  Fields will need to be re-added for the data to be saved, meaning it will require a re-import.
```

Comment:
How much effort would it take to save them properly as part of the initial import?

Reply:
Future initial imports will work, this is applies to pre-existing albums. I reworded the changelog entry, let me know if it's clearer.

I also realized that `mbsync` isn't enough, so removed that bit.

Reply:
Thanks for rewording - it is indeed more clear now 😃